javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,163 @@
+
+const weekDays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const weekDay = new Date();
+
+function getEventWeekday(eventday) {
+
+    weekDay.setDate(weekDay.getDate() + eventday);
+    return weekDays[weekDay.getDay()];
+}
+console.log(`Event held on ${getEventWeekday(6)}`);
+
+// Task 3  Weather to Wear cloth.
+
+const winter = weatherToWear(5);
+const spring = weatherToWear(14);
+const summer = weatherToWear(20);
+
+function weatherToWear(temperture) {
+    if (temperture <= 10) {
+        return "Sweter, Jacket & Warm Cap";
+    } else if (temperture <= 18) {
+        return "Fullsleve T-shirt & Jense";
+    } else {
+        return "short and a t-shirt ";
+    }
+}
+
+weatherToWear();
+console.log(winter);
+console.log(spring);
+console.log(summer);
+
+// Task 4 Student Manager 
+const class07Students = [];
+function addStudentToClass(studentName) {
+    // You write code here
+
+    if (class07Students.length > 6) {
+        if (studentName == "Queen") {
+            class07Students.push(studentName);
+        } else {
+            console.log("Cannot add more students to class");
+        }
+    } else {
+        if (class07Students.includes(studentName)) {
+            console.log("Student is already in the class");
+        } else {
+            class07Students.push(studentName);
+        }
+    }
+}
+
+addStudentToClass("Prakash");
+addStudentToClass("Prakash");
+addStudentToClass("Michel");
+addStudentToClass("Danial");
+addStudentToClass("Jakub");
+addStudentToClass("Milo");
+addStudentToClass("Tommy");
+addStudentToClass("Christian");
+addStudentToClass("Queen");
+addStudentToClass("Prakash");
+
+console.log(class07Students);
+
+function getNumberOfStudents() {
+    // You write code here
+
+    console.log(`Total Number of Student in Class ${class07Students.length}`);
+}
+getNumberOfStudents();
+
+// Task 5 Candy helper. 
+const boughtCandyPrices = []
+
+function addCandy(candyType, weight) {
+
+    if (candyType === "Sweet") {
+
+        boughtCandyPrices.push(weight * 0.5);
+
+    }
+
+    else if (candyType === "Chocolate") {
+
+        boughtCandyPrices.push(weight * 0.7);
+
+    }
+    else if (candyType === "Toffe") {
+        boughtCandyPrices.push(weight * 1.1);
+
+    }
+
+    else {
+        boughtCandyPrices.push(weight * 0.03);
+    }
+
+    return boughtCandyPrices;
+
+}
+addCandy("Sweet", 20);
+addCandy("Chocolate", 25);
+addCandy("Toffe", 15);
+addCandy("Chewing-gum", 40);
+console.log(boughtCandyPrices);
+
+// create function canBuyMoreCandy and match total price with math.random()
+const amountToSpend = Math.random() * 100;
+let total = 0;
+function canBuyMoreCandy() {
+
+    for (let i = 0; i < boughtCandyPrices.length; i++) {
+
+        total += boughtCandyPrices[i]
+
+    }
+    if (total <= amountToSpend) {
+        console.log("You can buy more, so please do!")
+    }
+    else {
+        console.log("Enough candy for you!")
+    }
+
+}
+canBuyMoreCandy();
```

Comment:
You can also check if it is possible to buy more candies before pushing to the array ie it should be in the `boughtCandyPrices` only if you have money for it (`amountToSpend`)